### PR TITLE
Channel transfer guides

### PR DESCRIPTION
**Pull request by alavers**
I've "rebranded" the existing guide to Sue Purb/Acme Bank and updated screenshots.

I've also added details to cover transferring to mobile app and tranfering a conversation to Facebook Messenger.

### CODE REVIEW

File: pages/guide/channel-transfer.md [hunk 1]
```diff
@@ -42,40 +53,173 @@ Smooch.init({
...
 ```
 
-By including this authCode in the init call Ellen is connected to the existing conversation that she had started on Facebook. When Ellen sends messages from the web, the help desk receives webhook events from Smooch indicating that Ellen is now using a new channel.
+![channel transfer to web](/images/channel_transfer_2.png)
+
+### Mobile
+
+Acme Bank has configured the URL `https://acme-bank.com` as a [universal link](https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html) on iOS and as an [app link](https://developer.android.com/training/app-links/index.html) on Android. In this way, when a mobile user taps this link they will be transferred to Acme Bank's native app if they have it installed.
+
+In their iOS banking app, Acme Bank adds a handler to their app delegate to run when Sue taps this link. First, the handler extracts the for consuming a Smooch auth code from the URL fragement (`#ac_9bkeBrqaAA1zkf4fy4slqmqU`) and uses it to initialize Smooch like so:
```
Comment (mspensieri) [inline pages/guide/channel-transfer.md]: > the handler extracts the for consuming a Smooch auth code from the URL fragement

wut?

File: pages/guide/channel-transfer.md [hunk 2]
```diff
@@ -42,40 +53,173 @@ Smooch.init({
...
+}
+```
 </aside>
 
+<aside class="subsection">
+When the SDK is initialized with an authCode, Sue is now connected to the existing conversation that she had started on Facebook. The helpdesk software will then recieve a <a href="https://docs.smooch.io/rest/#client-events">client:add webhook</a> to indicate that Sue has now connected to the conversation on a new channel.
+</aside>
 
-Ellen now sees the flight officer dashboard but she hasn't yet logged in. She does however see the history of messages that were sent between her and Ash over Facebook replicated here.
+Sue is now in the Acme baking app, but she hasn't yet logged in. She does however see the history of messages that were sent between her and Ash over Facebook replicated here.
```
Comment (mspensieri) [inline pages/guide/channel-transfer.md]: > baking

File: pages/guide/channel-transfer.md [hunk 3]
```diff
@@ -42,40 +53,173 @@ Smooch.init({
...
+        "primary": true,
+        "platform": "messenger",
+        "active": true
+    }]
+}
+```
+In response to this webhook, the help desk software notifies Ash that Sue has linked a new Facebook Messenger client.
+</aside>
+
+Ash sends Sue another message to let her know she can continue the conversation over Facebook Messeenger. Sue sees this message in Facebook Messenger while the conversaion log continues to be replicated in her browser chat window.
```
Comment (mspensieri) [inline pages/guide/channel-transfer.md]: > conversaion

Comment (alavers) [reply pages/guide/channel-transfer.md]: There are actually a lot of spelling errors now that I've bothered to turn on spell check

File: pages/guide/channel-transfer.md (hunk 1 above)
Comment (alavers) [reply pages/guide/channel-transfer.md]: Will fix!

File: pages/guide/channel-transfer.md [hunk 4]
```diff
@@ -6,20 +6,29 @@ layout: two-column
 
 # Channel Transfer
 
-Smooch facilitates transfering a user's conversation that began on a third party channels such as Facebook messenger into your own web site or mobile app. The user's conversation history is preserved, and the user is presented as a unified identity to your software backend. The guide below demonstrates how this works.
+Smooch facilitates transferring a user's conversation between Smooch SDKs and OTT channels like Facebook messenger. The user's conversation history is preserved, and the user is presented as a unified identity to your software backend. The guides below cover how to achieve this transfer in either direction:
 
-Ellen the flight officer is looking for a new contract at the Weyland Yutani Corporation. She first reaches out on their Facebook page. Weyland Yutani's Facebook page is integrated with Smooch, so Ellen's conversation shows up as a new conversation in their messaging system.
+- [Transfer to Web & Mobile](/guide/channel-transfer/#transfer-to-web-mobile)
+- [Transfer to Facebook Messenger](/guide/channel-transfer/#transfer-to-facebook-messenger)
```
Comment (jpjoyal) [inline pages/guide/channel-transfer.md]: Suggest calling that section "Transfer to chat apps" and inside the section mention facebook is used as an example but this list of channel supports it too

File: pages/guide/channel-transfer.md [hunk 5]
```diff
@@ -6,20 +6,29 @@ layout: two-column
 
 # Channel Transfer
 
-Smooch facilitates transfering a user's conversation that began on a third party channels such as Facebook messenger into your own web site or mobile app. The user's conversation history is preserved, and the user is presented as a unified identity to your software backend. The guide below demonstrates how this works.
+Smooch facilitates transferring a user's conversation between Smooch SDKs and OTT channels like Facebook messenger. The user's conversation history is preserved, and the user is presented as a unified identity to your software backend. The guides below cover how to achieve this transfer in either direction:
 
-Ellen the flight officer is looking for a new contract at the Weyland Yutani Corporation. She first reaches out on their Facebook page. Weyland Yutani's Facebook page is integrated with Smooch, so Ellen's conversation shows up as a new conversation in their messaging system.
+- [Transfer to Web & Mobile](/guide/channel-transfer/#transfer-to-web-mobile)
```
Comment (jpjoyal) [inline pages/guide/channel-transfer.md]: Nit: Add "SDK" after mobile

File: pages/guide/channel-transfer.md (hunk 4 above)
Comment (alavers) [reply pages/guide/channel-transfer.md]: Agree, do we have a more consistent term for this than "chat app"? The term doesn't currently occur anywhere in existing docs.

What about the term "messaging app"?

File: pages/guide/channel-transfer.md [hunk 6]
```diff
@@ -42,40 +53,212 @@ Smooch.init({
...
 
-![channel transfer to web](/images/channel_transfer_2.png)
+On Ash's end, the conversation continues within the same help desk context as before and the help desk indicates that Sue is now talking to him via the Acme banking app. Ash directs Sue to log in.
 
-On Ash's end, the conversation continues within the same help desk context as before and the help desk indicates that Ellen is now talking to him via the flight officer dashboard. Ash directs Ellen to log in to the dashboard using her flight officer credentials.
 <aside class="subsection">
-<p>The flight officer dashboard validates Ellen's credentials. It also <a href="/guide/authenticating-users/">issues a JWT</a> for Ellen and includes it in the dashboard login response. In the browser, the dashboard logs Ellen in to Smooch using this JWT.</p>
+<p>The banking app validates Sue's login credentials. In doing so, Acme Bank <a href="/guide/authenticating-users/">issues a JWT</a> for Sue and includes it in the dashboard login response. In the browser, the dashboard logs Sue in to Smooch using this JWT.</p>
```
Comment (jpjoyal) [inline pages/guide/channel-transfer.md]: suggest : "validates Sue's login credentials" -> "validates Sue's Acme bank account credentials"

File: pages/guide/channel-transfer.md [hunk 7]
```diff
@@ -6,20 +6,29 @@ layout: two-column
 
 # Channel Transfer
 
-Smooch facilitates transfering a user's conversation that began on a third party channels such as Facebook messenger into your own web site or mobile app. The user's conversation history is preserved, and the user is presented as a unified identity to your software backend. The guide below demonstrates how this works.
+Smooch facilitates transferring a user's conversation between Smooch SDKs and OTT channels like Facebook messenger. The user's conversation history is preserved, and the user is presented as a unified identity to your software backend. The guides below cover how to achieve this transfer in either direction:
```
Comment (jpjoyal) [inline pages/guide/channel-transfer.md]: Suggest a bit more context for the intro :

"Smooch can facilitates transferring a user's conversation from any Smooch channel to our SDKs, Facebook messenger, Telegram, {name all that support transfers}. By leveraging the linkRequest and authCode API, links can be created and sent as part of a conversation to direct a user to the channel of your choice. The user's conversation history is fully preserved, and the user is presented as a unified identity to your software backend. 

The guides below cover how to achieve this transfer in either direction:"

Comment (alavers) [reply pages/guide/channel-transfer.md]: I feel `{name all that support transfers}` makes the introductory sentence too long and begs to be outdated. The REST docs we link to list out which channels are supported and how.

Comment (alavers) [reply pages/guide/channel-transfer.md]: The rest makes sense though

**Pull request merged**